controllers/front/ProductController.php

```diff
@@ -328,6 +328,31 @@ public function initContent()
             // Assign attribute groups to the template
             $this->assignAttributesGroups($product_for_template);
         }
+
+        if (Tools::getValue('productajax')) {
+            ob_end_clean();
+            header('Content-Type: application/json');
+            die(json_encode([
```

Comment:
...here die... we should use die or ajaxDie? I think we should look for some unification